### PR TITLE
Define some Protobuf structures for starlake engine

### PR DESCRIPTION
**Pull request by sduzh**
## What type of PR is this：
- [ ] bug
- [ ] feature
- [ ] enhancement
- [ ] refactor
- [x] others

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage check]


:heart_eyes: **pass** : 0 / 0 (0%)

File: gensrc/proto/starlake.proto [hunk 1]
```diff
@@ -0,0 +1,59 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+syntax="proto2";
+
+package starrocks.starlake;
```
Comment (imay) [inline gensrc/proto/starlake.proto:5]: how about starrocks.lake?

File: gensrc/proto/starlake.proto [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+option java_package = "com.starrocks.proto.starlake";
```
Comment (imay) [inline gensrc/proto/starlake.proto:6]: com.starrocks.lake.proto?

File: gensrc/proto/starlake.proto (hunk 1 above)
Comment (sduzh) [reply gensrc/proto/starlake.proto]: updated

File: gensrc/proto/starlake.proto (hunk 2 above)
Comment (sduzh) [reply gensrc/proto/starlake.proto]: updated

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Review by imay — APPROVED** [review]

**Review by wyb — APPROVED** [review]

**Pull request merged**